data/json/mapgen/s_fabricstore.json

```
@@ -0,0 +1,91 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "s_fabricstore" ],
+    "//": "Frabic Outlet",
```

```suggestion
    "//": "Fabric Outlet",
```
Pretty obvious from the id anyway but